specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2020-11-20/examples/WorkbookTemplateAdd.json

```diff
@@ -0,0 +1,76 @@
+{
+  "parameters": {
+    "api-version": "2020-11-20",
+    "resourceGroupName": "my-resource-group",
+    "resourceName": "testtemplate2",
+    "subscriptionId": "subid",
+    "workbookTemplateProperties": {
+      "location": "west us",
+      "properties": {
+        "priority": 1,
+        "author": "Contoso",
+        "galleries": [
+          {
+            "name": "Simple Template",
+            "category": "Failures",
+            "type": "tsg",
+            "resourceType": "microsoft.insights/components",
+            "order": 100
+          }
+        ],
+        "templateData": {}
```

Comment:
Can you add an example for "templateData" property?

Reply:
"templateData" is the workbook template itself. I'll add an example